backend/src/main/java/org/flowacademy/hu/market/app/repositories/MarketRepository.java

```diff
@@ -14,4 +14,7 @@
 
     @Query("SELECT markets FROM Market markets WHERE markets.closingDate > UNIX_TIMESTAMP()")
     List<Market> findAllUpcomingMarkets();
+
+    @Query("SELECT markets FROM Market markets WHERE markets.closingDate > UNIX_TIMESTAMP()")
+    List<Market> findAllUpcomingMarketsForVendor();
```

Comment:
Itt a method nev nem stimmel. Ez a query semmit nem csinal a vendorokkal. A vendorokra szures a servicebe tortenik kulon lekeressel. Vagy keruljon be a querybe a vendor id, amivel szurtok ra, vagy nevezzetek at a method-ot.